docker-compose.yml

```diff
@@ -33,7 +33,7 @@ services:
       TEST_REDIS_HOST: redis
       TEST_REDIS_OLD_HOST: redis_old
       TEST_REDIS_OLD_PORT: 6379
-      DDTRACE_CI: true
+      DDTRACE_CI: 'true'
```

Comment:
GitHub Actions's Docker runner correctly complained that environment variables should not be native boolean values. For some reason, the local docker runners for MacOS does not complain about this.